dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageErrorMissingClass.java

```diff
@@ -17,23 +17,20 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import com.google.auto.value.AutoValue;
-
 /**
  * A missing class linkage error.
  */
 @AutoValue
 abstract class LinkageErrorMissingClass {
-  abstract String getTargetClassName();
-  abstract String getSourceClassName();
+  abstract ClassSymbolReference getReference();
```

Comment:
Do we still need the "LinkageError" classes at all? Are they each just going to hold onto one missing reference, or do you think they'll acquire other properties?